### PR TITLE
Change logo ministerie vws

### PR DESCRIPTION
**Pull request by MarinkeDefacto**

### CODE REVIEW

File: data/clients.yml [hunk 1]
```diff
@@ -110,7 +110,7 @@
   logo: "references/sogeti.png"
 
 - client: "Ministerie van Volksgezondheid, Welzijn en Sport"
-  logo: "references/ministerie-vws.png"
+  logo: "references/ministerie-vws-nieuw.png"
```
Comment (sn3p) [inline data/clients.yml]: Why not replace the old one?

**Comment by sn3p** [comment]
Can you push this to staging so I can see it in action?
See https://github.com/DefactoSoftware/website#stage-and-build-before

**Comment by MarinkeDefacto** [comment]
I have done a staging on the NL website.

**Comment by sn3p** [comment]
> I have done a staging on the NL website.

So https://staging.defacto.nl/klanten/ should have the logo? I have the feeling it's not updated yet.

**Comment by RonaldDijkstra** [comment]
Overruled by deployin #1151 to staging (don't touch!)

**Comment by MarinkeDefacto** [comment]
Removed the ministerie-vws-nieuw. I don't understand what you mean byoverruled by deployin #1151

**Comment by MarinkeDefacto** [comment]
Is this one oke to merge now?

**Pull request merged**